From aad52cc5facdcbaba65272e0f627263bd13363fe Mon Sep 17 00:00:00 2001
From: QiuJi <qiuji@iscas.ac.cn>
Date: Tue, 13 Apr 2021 11:17:53 +0800
Subject: [PATCH] [riscv64] Change one of the owners for RISCV.

Change-Id: I34d31032b8f1704ddea6334398a4ff2198cc192d
Reviewed-on: https://chromium-review.googlesource.com/c/v8/v8/+/2822631
Reviewed-by: Brice Dobry <brice.dobry@futurewei.com>
Reviewed-by: Hannes Payer <hpayer@chromium.org>
Commit-Queue: Brice Dobry <brice.dobry@futurewei.com>
Cr-Commit-Position: refs/heads/master@{#74069}
---
 RISCV_OWNERS | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/RISCV_OWNERS b/RISCV_OWNERS
index f3240b500b..8f8e15a40a 100644
--- a/RISCV_OWNERS
+++ b/RISCV_OWNERS
@@ -1,3 +1,3 @@
 brice.dobry@futurewei.com
-lazyparser@gmail.com
 peng.w@rioslab.org
+qiuji@iscas.ac.cn
-- 
2.35.1

